spec/models/scope_limitation_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe ScopeLimitation, type: :model do
+  let(:scope_limitations_populator) { ScopeLimitationsPopulator.new }
+
+  before do
+    allow(ScopeLimitationsPopulator).to receive(:new).and_return(scope_limitations_populator)
+  end
+
+  describe '.populate' do
+    it 'calls the scope_limitations_populator service' do
+      expect(scope_limitations_populator).to receive(:populate).with(no_args)
```

Comment:
If you add the class method like on my suggestion, you could just write `expect(ScopeLimitationsPopulator).to receive(:call).with(no_args)` and you wouldn't need the extra `let(:scope_limitations_populator) { ScopeLimitationsPopulator.new }`